accelerator/main.c

```diff
@@ -102,6 +103,11 @@ int main(int argc, char* argv[]) {
   pthread_t thread;
   pthread_create(&thread, NULL, health_track, (void*)&ta_core);
 
+  if (cli_mode) {
+    pthread_t thread2;
```

Comment:
Can we give these 2 thread a more meaningful name?

Reply:
I'll rename them to `health_thread` and `cli_thread`

Reply:
Nice